src/gpu.cpp

```diff
@@ -746,6 +746,10 @@ int create_gpu_instance()
                 if (physicalDeviceProperties.vendorID == 0x8086) // intel
                     gpu_info.subgroup_size = 32;
             }
+
+            // sanitize some wired subgroup size
```

Comment:
Hmm, it seems to be `weird` instead of `wired`?

Reply:
good catch !